src/index.ts

```diff
@@ -19,29 +19,37 @@ import {
   cancel,
   showCustomizationMessage,
 } from './commands';
-import { getConfig, findFiles } from './utils/vscode-extensions';
+import { getConfig, findFiles, asRelativePath } from './utils/vscode-extensions';
 
 function initialize(context: vscode.ExtensionContext) {
   const config = getConfig().vsicons;
   const settingsManager = new SettingsManager(vscode);
   registerCommands(context);
   manageWelcomeMessage(settingsManager);
   detectProject(findFiles, config)
-    .then((path) => {
-      if (path != null) {
-        const project = fs.readFileSync(path, "utf8");
-        const projectJson = (typeof project === "string") ? JSON.parse(project) : null;
-        if (projectJson) {
+    .then((results) => {
+      if (results != null && results.length) {
+        const isInRootFolder = asRelativePath(results[0].fsPath).includes('/');
```

Comment:
Update: After I corrected the condition check for the `isInRootFolder` (yeah, I made an `oops` there) I  packaged and installed the extension via `vsix`. Everything seems to work just fine.